features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/responsive/templates/sso/filter/template.jag

```diff
@@ -54,15 +50,24 @@
             if(site.ssoConfiguration.keyStoreName == null || site.ssoConfiguration.keyStoreName == ""){
                 site.ssoConfiguration.keyStoreName = CarbonUtils.getServerConfiguration().getFirstProperty("Security.KeyStore.Location");
                 }
+            if(site.ssoConfiguration.signRequests == null || site.ssoConfiguration.signRequests == ""){

```

Comment:
Fixed in #1494 (https://github.com/wso2/carbon-apimgt/commit/a031c40e48d28aafc715abc7fb45ae7ffbb48b9b)